Assets/LeapMotionModules/UI/Interaction/InteractionHand.cs

```diff
@@ -0,0 +1,780 @@
+﻿using Leap.Unity.RuntimeGizmos;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Leap.Unity.UI.Interaction {
+
+  public class InteractionHand {
+
+    public InteractionManager interactionManager;
+
+    private Func<Hand> _handAccessor;
+    private Hand _hand;
```

Comment:
You need to be careful when holding onto references to hands.  They can change out from underneath you due to the pooling done by the provider.  If you need the hand to stick around past fixedUpdate, make a copy of it.  If you don't, I would recommend passing the hand around via method arguments to make it very clear that this hand is not a long-term-field.